library/src/G4CMPImpactTunlNIEL.cc

```diff
@@ -48,7 +48,7 @@ PartitionNIEL(G4double energy, const G4Material *material,G4double Zin,
   const G4double A = GetEffectiveA(material);
 
   // Check if the material is silicon or similar
-  if (std::abs(Z-SiZ) < 0.5 || std::abs(A/(g/mole)-SiA) < 0.5) {
+  if (std::abs(Z-SiZ) < 0.5 || std::abs(A/(g/mole)-SiA) < 0.5)  {
```

Comment:
Why did you add an extra space here?  Please remove it, and this file should not need to be merged.